NOTICE.txt

```diff
@@ -5866,218 +5866,6 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
 
 
---------------------------------------------------------------------------------
-Dependency : github.com/elastic/ecs
```

Comment:
Is this part totally removed now, or is it just github that messes it up?

Reply:
Yes, it's removed because it's not available anymore in the ECS repo. We could have left the dependency pointing to v1.12 but eventually they could diverge, forcing us to fork anyway.